include/spore/codegen/codegen_cache.hpp

```diff
@@ -79,12 +91,16 @@ namespace spore::codegen
     {
         json["file"] = value.file;
         json["hash"] = value.hash;
+        json["file_size"] = value.file_size;
+        json["last_write_time"] = value.last_write_time;
     }
 
     void from_json(const nlohmann::json& json, codegen_cache_entry& value)
     {
         json["file"].get_to(value.file);
         json["hash"].get_to(value.hash);
+        json["file_size"].get_to(value.file_size);
```

Comment:
If the entry does not have it (old cache version), this will crash. We should be graceful about it.